salad/cli.py

```diff
@@ -8,37 +9,34 @@
 from salad.terrains.everything import *
 
 BROWSER_CHOICES = [browser.lower()
-                   for browser in DesiredCapabilities.__dict__.keys()
+                   for browser in list(DesiredCapabilities.__dict__.keys())
```

Comment:
this was part of making it compatible for python 3, but i think we can do without the .keys() and it will do the same